packages/components/input/src/input.ts

```diff
@@ -159,6 +159,12 @@ export const inputProps = buildProps({
     type: Boolean,
     default: true,
   },
+  /**
+   * @description maxlength
```

Comment:
It is recommended to keep the description consistent with the documentation.
`the max length`